tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "outDir": "./build"
   },
   "exclude": ["**/*.spec.ts", "**/*.spec.tsx"],
-  "include": ["packages"]
+  "include": [".eslintrc.js", "packages"]
```

Comment:
just a doubt: why do we need to add config file to the compilation list ??

Reply:
Since the eslintrc is not a js module any more this is no longer needed. 
Previously it would throw error "parserOptions.project" has been set for @typescript-eslint/parser" without this setting.